Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
 # Use mysql2 as the database for Active Record
 gem 'mysql2', '>= 0.4.4'
 # Use Puma as the app server
+
```

Comment:
in future if you need to add a gem, add a comment above it, currently you have added  autoprefixer gem below web-server comment